docs/python_docs/python/tutorials/getting-started/gluon_from_experiment_to_deployment.md

```diff
@@ -267,8 +267,7 @@ finetune_net.export("flower-recognition", epoch=epochs)
 MXNet provides various useful tools and interfaces for deploying your model for inference. For example, you can use [MXNet Model Server](https://github.com/awslabs/mxnet-model-server) to start a service and host your trained model easily.
 Besides that, you can also use MXNet's different language APIs to integrate your model with your existing service. We provide [Python](/api/python.html),    [Java](/api/java.html), [Scala](/api/scala.html), and [C++](/api/cpp) APIs.
 
-Here we will briefly introduce how to run inference using Module API in Python. There is more detailed explanation available in the [Predict Image Tutorial](https://mxnet.apache.org/tutorials/python/predict_image.html).
-In general, prediction consists of the following steps:
+Here we will briefly introduce how to run inference using Module API in Python. In general, prediction consists of the following steps:
```

Comment:
This is the broken link discussed in #16724

Reply:
Do you want to add it back? or the image-classification shouldn't exist anymore? What's the solution for that issue?

Reply:
@ChaiBapchya #16724 is still open, I wondered if something is pending. I personally don't know the expect solution, if everything is good we can close it.

But following link was mentioned in the issue, not sure if this need to be considered.
> True link should be -- https://github.com/dmlc/mxnet-notebooks/blob/master/python/tutorials/predict_imagenet.ipynb

Reply:
I see there was a link which previously existed. Now the link has been removed. Have a word with Aaron/Talia they have a better idea.